.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a bug report for the Next.js core / examples
```

Comment:
```suggestion
about: Create a bug report for the SWR library
```

Reply:
Great great catch!

Reply:
now you know this line is from next.js repo 🙈